### DISCUSSION TITLE
Left-align long commands

### DISCUSSION THREAD

**Discussion by rieje** [junegunn/fzf#3170] [Q&A] [answered]
With Ctrl-R, some really long commands (e.g. aria2c commands) that cannot be shown in a full line for some reason shows only the middle of the line. This makes the results very confusing--is there a way to make the result items left-aligned so I always see what the actual command is in the results? It could very well be e.g. a curl command.

I already have a key bound to show preview of the full command. I just want the results to be useful by showing the beginning of the command for useful context, e.g. what program is the command running. It's strange because the items are neither left nor right-aligned in my case. Actually, I just tested it and it seems it's left-aligned if I make the window as big as possible (the window I was having problems with wasn't particularly small).

**Comment by junegunn** [comment, marked as answer]
```sh
export FZF_CTRL_R_OPTS=--no-hscroll
```
